dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/partition/JdbcPartitionManager.java

```diff
@@ -65,8 +65,12 @@ public Set<String> getAnalyticsPartitions( AnalyticsTableType tableType )
             return analyticsPartitions.get( tableType );
         }
 
+        // TODO: Temporary fix to no break the application after analytics
```

Comment:
It's needed so we can test this branch correctly.
This issue was introduced recently, and this `workaround` should be removed as soon as the issue is correctly fixed.